### DISCUSSION TITLE
Synchronize tables by projects

### DISCUSSION THREAD

**Discussion by maxrodro** [Mimetis/Dotmim.Sync#653] [Q&A]
Hello everyone, I have the following case:

I have a database with X number of tables for example:
dbo.Table1
dbo.Table2
dbo.Table3
dbo.Table4
dbo.Table5

I have 2 projects that synchronize certain tables for example
Project 1
           **Table1**   Type: DownloadOnly
           Table2         Type: DownloadOnly
           Table3         Type: DownloadOnly

Project 2
           **Table1** Type: UploadOnly
           Table4       Type: DownloadOnly
           Table5       Type: DownloadOnly

How can I do a synchronization of these tables per project in Dotmim?

**Comment by Mimetis** [comment]
Not sure, but I guess you can use [Multi Scopes](https://dotmimsync.readthedocs.io/MultiScopes.html) 
I did not tested it, but it can be a potential solution (but once again, never tested it with your scenario)

**Reply by maxrodro** [reply]
Thank you very much @Mimetis , I'll try
